lib/ClangImporter/ImporterImpl.h

```diff
@@ -516,7 +516,7 @@ class LLVM_LIBRARY_VISIBILITY ClangImporter::Implementation
   std::unique_ptr<clang::Parser> Parser;
 
   /// Clang parser, which is used to load textual headers.
-  std::unique_ptr<clang::MangleContext> Mangler;
+  std::shared_ptr<clang::MangleContext> Mangler;
```

Comment:
Why is this necessary?

Reply:
I reverted this to `unique_ptr` and replaced with something simpler.

We're now reusing `ClangImporter::Implementation::getMangledName` to create an Itanium-mangled name for the synthesized Swift function that generates the default argument. We can't reuse `ClangImporter::Implementation::Mangler` though, because it might not be an Itanium mangler, which would produce a name that isn't a valid identifier in Swift.